Makefile

```diff
@@ -26,7 +26,7 @@ TIMEOUT = 60
 
 .PHONY: precommit ci
 precommit: license-check misspell go-mod-tidy golangci-lint-fix test-default
-ci: dependabot-check license-check lint vanity-import-check build test-default check-clean-work-tree test-coverage
+ci: dependabot-check license-check lint vanity-import-check crosslink build test-default check-clean-work-tree test-coverage
```

Comment:
I think this should probably just be part of the build target instead.  Like, `generate` the proper `replace`s should exist before trying to build.

Reply:
I've moved it there.